src/realtimecontroller/RealtimeController.ts

```diff
@@ -58,6 +58,11 @@ export default interface RealtimeController {
     callback: (attendeeId: string, present: boolean, externalUserId?: string | null) => void
   ): void;
 
+  /**
+   * Retrieve the externalUserId from the attendeeId-externalUserId map
```

Comment:
The fact that it is store in a map is an internal detail that does not need to be mentioned in the API method. Suggest: "Returns the external user ID for a given attendee ID"

Reply:
changed the comment